buildenv/jenkins/docker-slaves/ppc64le/centos7/Dockerfile

```diff
@@ -95,7 +94,8 @@ RUN cd /usr/src/ \
   && cd autoconf-2.69 \
   && ./configure --build=$(rpm --eval %{_host}) \
   && make \
-  && make install
+  && make install \
+  && rm -rf /usr/src/autoconf-2.69
```

Comment:
Not sure if we should cd out of the directory before removing it? What is best practice?